duckduckgo/macos-browser
# Microsoft logo for address bar animation

Task/Issue URL: https://app.asana.com/0/1199230911884351/1202425934337331/f


**Description**:
Microsoft tracker icon for the address bar animation



**Steps to test this PR**:
1. Visit https://www.workplace.com/
2. Verify Microsoft animation icon is part of the address bar animation

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
